db/migrate/20230814171021_add_collection_notes_to_bulk_exports.rb

```diff
@@ -1,5 +1,5 @@
 class AddCollectionNotesToBulkExports < ActiveRecord::Migration[6.0]
   def change
-    add_column :bulk_exports, :collection_notes, :boolean
```

Comment:
Because we've already applied the earlier version of this code, we need a new migration that renames `collection_notes` to `notes_csv`.  When I try to do a bulk export on this branch, I get "undefined method `notes_csv' for #<BulkExport:0x00007f09953aeaa8>"

Reply:
Fixed 👍